oceannavigator/frontend/src/components/AreaWindow.jsx

```diff
@@ -597,7 +597,7 @@ class AreaWindow extends React.Component {
         plot_query.neighbours = this.props.options.interpNeighbours;
         plot_query.plotTitle = this.state.plotTitle;
         if (this.state.dataset_compare) {
-          plot_query.compare_to = this.props.dataset_1;
+          plot_query.compare_to = this.state.dataset_1;
```

Comment:
```suggestion
          plot_query.compare_to = this.props.dataset_1;
```

I think this was left over from the hack we used to get around the compare datasets issue we were having.